bitrader/arbitrage_tools.py

```diff
@@ -317,11 +317,11 @@ def arbitrage(amount, coin_code='XBT', coin_name='bitcoin', exchange_name='Luno'
             exchange_rate = get_forex_buy_quote('EUR')
 
         if transfer_fees:
-            _swift_fee = Decimal(105)
+            _swift_fee = Decimal(110)
             # TODO: Factor in ebucks?
-            _fnb_comission = min(max(transfer_amount * Decimal(0.0052), Decimal(140)), Decimal(650))
+            _fnb_comission = min(max(transfer_amount * Decimal(0.0055), Decimal(140)), Decimal(650))
             # TODO: With SEPA deposit fee is 0. But check other methods?
```

Comment:
You can remove the TODO about the SEPA fees, but maybe make add a comment with the link to the fees page on Kraken?